g2o/types/sba/types_six_dof_expmap.cpp

```diff
@@ -107,6 +118,89 @@ bool VertexSE3Expmap::write(std::ostream& os) const {
   return os.good();
 }
 
+VertexSE3ExpmapWriteGnuplotAction::VertexSE3ExpmapWriteGnuplotAction(): WriteGnuplotAction(typeid(VertexSE3Expmap).name()){}
+
+HyperGraphElementAction* VertexSE3ExpmapWriteGnuplotAction::operator()(HyperGraph::HyperGraphElement* element, HyperGraphElementAction::Parameters* params_){
+  if (typeid(*element).name()!=_typeName)
+    return 0;
+  WriteGnuplotAction::Parameters* params=static_cast<WriteGnuplotAction::Parameters*>(params_);
+  if (!params->os){
+    std::cerr << __PRETTY_FUNCTION__ << ": warning, no valid os specified" << std::endl;
+    return 0;
+  }
+  
+  VertexSE3Expmap* v =  static_cast<VertexSE3Expmap*>(element);
+
+  auto est = v->estimate().toMinimalVector();
+  for (int i=0; i<6; i++)
+    *(params->os) << est[i] << " ";
+  *(params->os) << std::endl;
+  return this;
+}
+
+#ifdef G2O_HAVE_OPENGL
+void drawTriangle(float xSize, float ySize){
```

Comment:
Is this the `g2o` namespace? This will cause a multiple symbol error if untreated since `vertex_se3.cpp` also has `void drawTriangle(float xSize, float ySize)`. Declare it `static` or put it in an specific or anonymous namespace.

Reply:
`drawTriangle` of `vertex_se3.cpp` (g2o master's), `vertex_se3_quat.cpp` (g2o master's) and `types_six_dof_expmap.cpp` (current branch) all lies in different shared objects.

Anyway I have added `static` to the `drawTriangle` which I added in `types_six_dof_expmap.cpp`